### ISSUE TITLE
QDeled from a chute

### ISSUE THREAD

**Issue by ArcLumin** [HippieStationArchive/HippieStation13-1#3394]
I entered a disposal as a ling, and when I did, I hit the edge. After getting that message, I was immediately qdeled. WHAT

**Comment by blobbernaut** [comment]
So fucking glad because of who you were that round, but yeah. No idea what could cause this.

**Comment by ArcLumin** [comment]
Why were you glad, did I succ you? :^)

**Comment by KayeArray** [comment]
I guess it decided to chute first

**Comment by ArcLumin** [comment]
top jokes my dude

**Comment by Spacedong** [comment]
what

**Comment by yobob591** [comment]
I thought you knew  NT has started powering disposals with micro singularities

**Comment by Pyko1** [comment]
Err.
You hit what edge?
Tried to exploit zLevels and hit the edge of map or what?

Or you hit your head so hard on to the edge of thrashcan when you tried to enter it qdel'ed you?

**Comment by ArcLumin** [comment]
Hit my head so hard on the edge that I qdeled. Or some shit. It was just a normal disposal on mining.

**Comment by Pyko1** [comment]
i wonder if it was missing chute under it or some shit

**Comment by ArcLumin** [comment]
Nah, I readmined myself and spawned myself as a test dummy to check, it didn't happen again. No issues were with the chute; I had come through the chute earlier even.
